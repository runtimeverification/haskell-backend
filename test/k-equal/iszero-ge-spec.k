requires "test-k-equal.k"

module ISZERO-GE-SPEC
  imports TEST-K-EQUAL

  claim <k> #iszero(I) => #done(1) </k> requires I >=Int 0
endmodule
