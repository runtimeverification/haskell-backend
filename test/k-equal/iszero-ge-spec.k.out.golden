  #Not ( {
    1
  #Equals
    bool2Int ( I ==Int 0 )
  } )
#And
  <T>
    <k>
      #done ( bool2Int ( I ==Int 0 ) ) ~> .K
    </k>
  </T>
#And
  {
    true
  #Equals
    I >=Int 0
  }
